tests/unit/server/data-access/MwBotWikibaseRepo.spec.ts

```diff
@@ -0,0 +1,188 @@
+import EntityInitializer from '@/common/EntityInitializer';
+import FingerprintableEntity from '@/datamodel/FingerprintableEntity';
+import MwBotWikibaseRepo from '@/server/data-access/MwBotWikibaseRepo';
+import EntityNotFound from '@/server/data-access/error/EntityNotFound';
+import TechnicalProblem from '@/server/data-access/error/TechnicalProblem';
+import mwbot from 'mwbot';
+
+function newMwBotWikibaseRepo( bot: any, initializer?: any ) {
```

Comment:
Do we need 'any' at bot here?

Reply:
What would be the appropriate type for "random mocks I make up at will"?